posthog/temporal/data_imports/pipelines/pipeline_sync.py

```diff
@@ -248,7 +255,17 @@ def _run(self) -> dict[str, int]:
                 total_counts = counts + total_counts
 
                 if total_counts.total() > 0:
-                    delta_tables = get_delta_tables(pipeline)
+                    # Fix to upgrade all tables to DeltaS3Wrapper
+                    resouce_names = list(self.source._resources.keys())
```

Comment:
guess this is a typo (i.e. should be `resource_names`) but given the urgency of this change I don't think this is important